draft-ietf-quic-transport.md

```diff
@@ -3855,9 +3857,7 @@ during connection establishment and when migrating to a new path
 On receiving a QUIC packet with an ECT or CE codepoint, an ECN-enabled endpoint
 that can access the ECN codepoints from the enclosing IP packet increases the
 corresponding ECT(0), ECT(1), or CE count, and includes these counts in
-subsequent ACK frames; see {{generating-acks}} and {{frame-ack}}.  Note that
-this requires being able to read the ECN codepoints from the enclosing IP
-packet, which is not possible on all platforms.
```

Comment:
I'm not sure about this one.  IIRC, we were pretty deliberate about adding this text to note that compliance with the ECN guidance was conditional on OS support for doing so.

Reply:
Yes, added in response to #2156 (PR #2201).

Reply:
Am fine with leaving this as is. @gorryfair?

Reply:
Simply ignoring ECN is not an acceptable position.  My comment here was not a request to remove that. It was a request to clarify what happens on those other platforms: "On those platforms, QUIC will not enable use of ECN for this endpoint.”

Reply:
But if the platform doesn't allow access to the ECN bits, the QUIC stack will not be able to use ECN *at all*. So why scope this to an endpoint?

Reply:
OK!  SO what I intended to be clear about was if a platform can't receive ECN marks, then use of ECN can't be enabled to this endpoint.

I too would like to leave the note.

Reply:
I added a paragraph to the previous section to clarify this.

Reply:
That new text has resolved this NiT.